### PR TITLE
Fix some typos in the README

### PR DESCRIPTION
**Pull request by dluc**

### CODE REVIEW

**Comment by dluc** [comment]
I tried to sign the CLA twice, couldn't get to the end of the process. Submitted the contact form, in case someone wants to look into it.

File: cluster-autoscaler/README.md [hunk 1]
```diff
@@ -54,19 +54,19 @@ depends on the cloud provider and the speed of node provisioning.
...
 if the node is deleted so they make a node needed, even if its utilization is low. While 
-checking this condition the new locations of all pods are memorized. With that Cluster Autoscaler
+checking this condition, the new locations of all pods are memorized. With that, Cluster Autoscaler
 knows where each pod can be moved and which nodes depend on which other nodes in terms of 
-pod migration. Of course, it may happen that eventaully the scheduler will place the pods 
+pod migration. Of course, it may happen that eventually the scheduler will place the pods 
 somewhere else. 
 
-* There are no kube-system pods on the node (except these that run on all nodes by default like 
+* There are no kube-system pods on the node (except theoe that run on all nodes by default like
```
Comment (mwielgus) [inline cluster-autoscaler/README.md]: theoe?

**Comment by mwielgus** [comment]
Thank you for fixing this. Almost LGTM - just a single comment.

**Comment by mumoshu** [comment]
Hi @dluc, any chance you could fix [the typo](https://github.com/kubernetes/contrib/pull/2362#discussion_r101513532) and resolving conflicts? Thanks!

**Comment by dluc** [comment]
Looks like the file is considerably smaller than the time when I sent the PR. Let me close this one, I sent a new PR https://github.com/kubernetes/contrib/pull/2557

**Pull request closed without merging**